### ISSUE TITLE
Application does not start with 0.8.1

### ISSUE THREAD

**Issue by colq2** [NativePHP/laravel#399]
### What were you trying to do?

I updated to newest version 0.8.1 and build a new version off the app, install it and run it.

### What happened?

App does not open. On windows it is in the task manager but there is no ui. On mac I get an error on startup:

**A JavaScript error occurred in the main process**

```
Uncaught Exception:
Error: spawn /Applications/xxxxx.app/Contents/Resources/app.asar.unpacked/resources/php/php ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

### How to reproduce the bug

Build Application, install it and start it.

### Package Versions

nativephp/electron@0.8.1, nativephp/laravel@0.5.7, nativephp/php-bin@0.5.4

### PHP Version

8.3.0

### Laravel Version

v11.26.0

### Node Version

v20.13.0

### Which operating systems have you seen this occur on?

macOS, Windows

### OS version

win11, macos 14.6.1

**Comment by simonhamp** [comment]
> nativephp/electron@0.8.1, nativephp/laravel@0.5.7

this is strange. you should be running `nativephp/laravel@0.6.1` here... i'm not clear why this wouldn't have updated.... did you run `composer update`?

**Comment by colq2** [comment]
Mysterious... I am pretty sure I did.
Now I removed vendor, cleared composer cache and installed again. I get now the right version and build it again. 
But I get the same error.

nativephp/electron@0.8.1
nativephp/laravel@0.6.1
nativephp/php-bin@0.5.4

Is there anything else I could try or provide more information for you?

**Comment by colq2** [comment]
I also tried a fresh laravel app with a fresh install of nativephp and get also the same error when I run the application. But it works with `php artisan nativephp:serve' in development.

**Comment by bee-interactive** [comment]
Hi there! Same issue for me, also with a fresh laravel 11 application, MacOS Sonoma, PHP 8.3 and the latest version of the NativePHP package. artisan native:serve works well. Only when I try to open the build version as a traditionnal mac application

**Comment by arislanhaikal** [comment]
Some issue for me
- laravel 11
- php 8.3
- nativephp/electron ^0.8.3
- macOs Sequoia

![000320-08fMy1je@2x](https://github.com/user-attachments/assets/416587cc-6161-4afc-ac06-6a09c12f180c)

**Comment by simonhamp** [comment]
Confirmed that I'm seeing the same.... I'm investigating

**Comment by JCombee** [comment]
I have looked into this myself since I ran into this as well.

What I uncovered was that the app path was not correctly set in the electron package.
I think this might be a remenant from moving the plugin from a node module to the composer package.

I deducted this from the full error message I recieved on this.

```
Could not open input file: artisan
PS C:\xxx\dist\win-unpacked> .\xxx.exe
PS C:\xxx\dist\win-unpacked> 
Error: spawn C:\xxx\dist\win-unpacked\resources\app.asar.unpacked\resources\php\php.exe ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn E:\\Herd\\xxx\\dist\\win-unpacked\\resources\\app.asar.unpacked\\resources\\php\\php.exe',
  path: 'E:\\Herd\\xxx\\dist\\win-unpacked\\resources\\app.asar.unpacked\\resources\\php\\php.exe',
  spawnargs: [ 'artisan', 'native:config' ],
  cmd: 'E:\\Herd\\xxx\\dist\\win-unpacked\\resources\\app.asar.unpacked\\resources\\php\\php.exe artisan native:config',
  stdout: '',
  stderr: ''
}
(node:20792) UnhandledPromiseRejectionWarning: TypeError: Error processing argument at index 0, conversion failure from undefined
    at Object.setAppUserModelId (C:\xxx\dist\win-unpacked\resources\app.asar\node_modules\@electron-toolkit\utils\dist\index.cjs:18:20)
    at NativePHP.setAppUserModelId (C:\xxx\dist\win-unpacked\resources\app.asar\out\main\index.js:1678:23)
    at NativePHP.<anonymous> (C:\xxx\dist\win-unpacked\resources\app.asar\out\main\index.js:1649:12)
    at Generator.next (<anonymous>)
    at fulfilled (C:\xxx\dist\win-unpacked\resources\app.asar\out\main\index.js:1579:24)
    at process.processTicksAndRejections (node:internal/process/tas^Cqueues:95:5)
```

There you see it crashed somewhere during `NativePHP.setAppUserModelId` which was the first function using the config data from the project. I placed a console.log to get the input data which showed me the `cwd` was not correct. So by adjusting the relative path to the application root I could fix this.

Here is the PR: https://github.com/NativePHP/electron/pull/125

**Comment by simonhamp** [comment]
Solved via https://github.com/NativePHP/electron/pull/125. Thanks @JCombee 🙏🏼 

Release coming tomorrow.

**Comment by simonhamp** [comment]
Fix released in [`nativephp/electron@0.8.4`](https://github.com/NativePHP/electron/releases/tag/0.8.4)

Thanks again @JCombee

**Comment by bee-interactive** [comment]
@simonhamp @JCombee  Thank you so much for the reactivity!
